app/src/org/commcare/activities/CommCareVerificationActivity.java

```diff
@@ -224,7 +224,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
         if (requestCode == GET_MULTIMEDIA && resultCode == Activity.RESULT_OK) {
             // we found some media, so try validating it
             newMediaToValidate = true;
+            return;
         }
+
+        super.onActivityResult(requestCode, resultCode, intent);

```

Comment:
From what I can tell, it is one of those things that it actually doesn't matter and is only worth putting in for the sake of extensibiltiy. For instance `Activity.onActivityResult` is an empty implementation.
I think `FragmentActivity.onActivityResult` dispatches to associated fragments, so in that sense it might be more important... 
I think if something is processed via the overriding version, it probably doesn't also need to be passed up to super classes for processing.